cmd/rpcdaemon/health/check_block.go

```diff
@@ -22,3 +25,45 @@ func checkBlockNumber(blockNumber rpc.BlockNumber, api EthAPI) error {
 
 	return nil
 }
+
+func checkBlockTime(api EthAPI) (*uint, error) {
+	if api == nil {
+		return nil, fmt.Errorf("no connection to the Erigon server or `eth` namespace isn't enabled")
+	}
+	fullTx := false
+	data, err := api.GetBlockByNumber(context.TODO(), -1, &fullTx)
```

Comment:
you could pass in the http request context here instead of using a todo and it would automatically be cancelled if the parent web request is closed early.